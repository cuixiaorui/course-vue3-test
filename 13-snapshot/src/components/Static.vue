<template>
  <div>
    <h1>hi my name is a static component b</h1>
  </div>
</template>

<script>
export default {};
</script>

<style></style>
