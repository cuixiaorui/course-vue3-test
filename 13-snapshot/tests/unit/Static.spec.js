import Static from "@/components/Static.vue";
import { shallowMount } from "@vue/test-utils";

describe("static", () => {
  test("snapshot ", () => {
    const wrapper = shallowMount(Static);
    expect(wrapper.element).toMatchSnapshot();
  });
});
