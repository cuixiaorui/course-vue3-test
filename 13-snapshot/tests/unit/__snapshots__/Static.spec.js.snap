// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`static snapshot  1`] = `
<div>
  <h1>
    hi my name is a static component b
  </h1>
</div>
`;
